include/linker/kobject-text.ld

```diff
@@ -32,11 +32,11 @@
 
 	ASSERT(
 		CONFIG_KOBJECT_TEXT_AREA >= _kobject_text_area_used,
-"The configuration system has incorrectly set
-'CONFIG_KOBJECT_TEXT_AREA' to
-CONFIG_KOBJECT_TEXT_AREA, which is not big enough. You must
-through Kconfig either disable 'CONFIG_USERSPACE', or set
-'CONFIG_KOBJECT_TEXT_AREA' to a value larger than
+"The configuration system has incorrectly set \
+'CONFIG_KOBJECT_TEXT_AREA' to \
+CONFIG_KOBJECT_TEXT_AREA, which is not big enough. You must \
+through Kconfig either disable 'CONFIG_USERSPACE', or set \
+'CONFIG_KOBJECT_TEXT_AREA' to a value larger than \
 CONFIG_KOBJECT_TEXT_AREA."
 		);
 #endif /* CONFIG_USERSPACE */
```

Comment:
see sanitycheck: "Please remove blank lines at end of 'include/linker/kobject-text.ld'"

Reply:
Hmm, it's strange why sanitycheck detects this as I don't change that part of file. However I's fixed now.